Cubical/HITs/SetTruncation/Fibers.agda

```diff
@@ -0,0 +1,294 @@
+{-
+
+This file contains:
+
+- Fibers of induced map between set truncations is the set truncation of fibers
+  modulo a certain equivalence relation defined by π₁ of the base.
```

Comment:
There are a lot of functions in this file with uninformative names, can you clean it up a bit?

Reply:
This part is my biggest headache. The eliminators (of truncations) have to be applied in an extremely twisted way. I wrote a whole bunch of helpers only to keep my sanity. One way to clean it up is to nest everything up, so we get fewer names. What do you think?

Reply:
Hm, that could be good if it doesn't end up making the code harder to read. I need to look at this file longer to understand what it's doing... maybe there is some chance for simplifying.

Reply:
I remember when I wrote this, the greatest problem is I got lost in figuring out where I should put all these eliminators...

Reply:
Think I found a way to shorten, I will hack on it a bit.